hack/windows/Install-OVS.ps1

```diff
@@ -5,22 +5,12 @@
   .PARAMETER DownloadURL
   The URL of the OpenvSwitch package to be downloaded.
 
-  .PARAMETER DownloadDir
-  The path of the directory to be used to download OpenvSwitch package. The default path is the working directory.
-
   .PARAMETER OVSInstallDir
   The target installation directory. The default path is "C:\openvswitch".
 
-  .PARAMETER CheckFileHash
-  Skips checking file hash. The default value is true.
-
   .PARAMETER LocalFile
   Specifies the path of a local OpenvSwitch package to be used for installation.
-  When the param is used, "DownloadURL" and "DownloadDir" params will be ignored.
-
-  .PARAMETER ImportCertificate
-  Specifies if a certificate file is needed for OVS package. If true, certificate
-  will be retrieved from OVSExt.sys and a package.cer file will be generated.
+  When the param is used, "DownloadURL" params will be ignored.
```

Comment:
```suggestion
  When the param is used, "DownloadURL" param will be ignored.
```

Reply:
Updated as `When the param is used, "DownloadURL" is ignored`